src/unix/linux-core.c

```diff
@@ -228,6 +228,7 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
     assert(w->fd >= 0);
     assert(w->fd < (int) loop->nwatchers);
 
+    memset(&e, 0, sizeof(e));
```

Comment:
You can move the memset out of the loop.  There is at least one more place in this file where epoll_ctl() is called without a scrubbed struct epoll_event (`uv__io_check_fd()`, there may be more.)